frontstage/controllers/notify_controller.py

```diff
@@ -0,0 +1,69 @@
+import logging
+from urllib import parse as urlparse
+
+import structlog
+
+from frontstage.exceptions import exceptions
+from requests_wrapper import Requests
+
+logger = structlog.wrap_logger(logging.getLogger(__name__))
+
+
+class NotifyGateway:
+    """ Client for Notify gateway"""
+
+    def __init__(self, config):
+        self.config = config
+        self.notify_url = config['RAS_NOTIFY_SERVICE_URL']
+        self.email_verification_template = config['RAS_NOTIFY_EMAIL_VERIFICATION_TEMPLATE']
+        self.request_password_change_template = config['RAS_NOTIFY_REQUEST_PASSWORD_CHANGE_TEMPLATE']
+        self.confirm_password_change_template = config['RAS_NOTIFY_CONFIRM_PASSWORD_CHANGE_TEMPLATE']
+        self.notify_account_locked = config['RAS_NOTIFY_ACCOUNT_LOCKED_TEMPLATE']
+
+    def _send_message(self, email, template_id, personalisation=None, reference=None):
+        """
+        Send message to gov.uk notify wrapper
+        :param email: email address of recipient
+        :param template_id: the template id on gov.uk notify to use
+        :param personalisation: placeholder values in the template
+        :param reference: reference to be generated if not using Notify's id
+        :rtype: 201 if success
+        """
+
+        if not self.config['SEND_EMAIL_TO_GOV_NOTIFY']:
```

Comment:
Can you do 3 changes to this.
- Add `self.send_email_to_notify = self.config['SEND_EMAIL_TO_GOV_NOTIFY']` to __init__
- Remove `self.config = config` from __init__
- Replace this line with `if not self.send_email_to_notify:`

This object gets the config passed to it, then it's saved as an attribute for seemingly no reason.  This'll just make it tidier and discourages people from being lazy with this object by doing `self.config['thing']` if they want something